lib/yardstick/rule_description/classifier.rb

```diff
@@ -0,0 +1,105 @@
+module Yardstick
+  # Rule description composed of tokens which can be formatted
+  class RuleDescription
+    # Token classification
+    module Classifier
+      # List of classifiers
+      class List
+        extend Forwardable
+        include Concord.new(:classifiers), Enumerable
+
+        # Classify a token by returning the first match
+        #
+        # @param token [String] markup text
+        #
+        # @return [Token]
```

Comment:
If no classifier matches the token, won't the return value be `classifiers`? Is that what you want or does it not matter in the places where `#classify` is used?